apps/frontend/src/libs/components/create-poi-modal/create-poi-modal.tsx

```diff
@@ -74,6 +58,12 @@ const CreatePOIModal = ({
 					name="name"
 					type="text"
 				/>
+				<MapLocationField<CreatePOIFormValues>
```

Comment:
Do we need to pass it here? for inputs its automatically inferred from the control
```suggestion
				<MapLocationField
```